torchsde/_core/methods/milstein.py

```diff
@@ -50,55 +49,47 @@ def v_term(self, I_k, dt):
         raise NotImplementedError
 
     @abc.abstractmethod
-    def y_prime_f_factor(self, dt, f_eval):
+    def y_prime_f_factor(self, dt, f):
         raise NotImplementedError
 
     def step(self, t0, t1, y0):
         dt = t1 - t0
         I_k = self.bm(t0, t1)
         v = self.v_term(I_k, dt)
 
-        f_eval = self.sde.f(t0, y0)
-        g_prod_eval = self.sde.g_prod(t0, y0, I_k)
+        f = self.sde.f(t0, y0)
+        g_prod_I_k = self.sde.g_prod(t0, y0, I_k)
 
         if self.options[METHOD_OPTIONS.grad_free]:
-            g_eval = self.sde.g(t0, y0)
-            g_prod_eval_v = self.sde.g_prod(t0, y0, v)
+            g = self.sde.g(t0, y0)
+            g_prod_v = self.sde.g_prod(t0, y0, v)
             sqrt_dt = torch.sqrt(dt) if isinstance(dt, torch.Tensor) else math.sqrt(dt)
```

Comment:
I think `dt` can only be a tensor now (as input times are now coerced to tensor), so this can be simplified.

Reply:
Done.